src/modules/core/commands/confess.ts

```diff
@@ -42,7 +42,8 @@ export default Command(
       .setTitle(`Confession`)
       .setDescription(confession);
 
-    if (isImageUrl(confession)) {
+    const imageUrl = await getImageUrl(confession);
+    if (imageUrl) {
       confessionEmbed.setImage(confession);
```

Comment:
This is probably supposed to be `imageUrl` now and not `confession`

Reply:
done